examples/rules_cc/hello_world_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
can we have a slightly different test that sets such flags:

 ````
 --experimental_use_hermetic_linux_sandbox \
    --sandbox_writable_path="$cache_prefix" \
    --sandbox_add_mount_pair=/proc \
```

? it doesn't have to print anything; maybe just returning `42` is enough.

Reply:
IMO this example should be as simple as possible, so that our users don't have to think much and can just essentially copy/paste a minimel example. Having separate tests with a full configuration would be nice though.